src/Analyzer/ConfluenceAnalyzer.php

```diff
@@ -112,8 +161,14 @@ public function analyze( SplFileInfo $file ): bool {
 			return true;
 		}
 
+		if ( isset( $this->config['config'] ) ) {
+			$this->advancedConfig = $this->config['config'];
+			$this->hasAdvancedConfig = true;
```

Comment:
... especially as there does not seem to be code to set it to `false`...

Reply:
done